src/commands/git-metadata/upload.ts

```diff
@@ -213,5 +213,5 @@ UploadCommand.addPath('git-metadata', 'upload')
 UploadCommand.addOption('dryRun', Command.Boolean('--dry-run'))
 UploadCommand.addOption('verbose', Command.Boolean('--verbose'))
 UploadCommand.addOption('directory', Command.String('--directory'))
-UploadCommand.addOption('gitSync', Command.Boolean('--git-sync'))
```

Comment:
What happens for the users using this argument? Would it break?

Reply:
very good point, I'll need to add the support for this arg back. Thanks!